### DISCUSSION TITLE
专业版突然变成免费版，请帮忙恢复

### DISCUSSION THREAD

**Discussion by casperwzy** [getlantern/lantern-forum-cn#277] [Q&A] [answered]
3月26日，不知道为什么专业版就突然变成免费版了，今天仍然是免费版，相关的付款凭证，和账号邮箱已经发送电子邮件至 [ [support@getlantern.org](mailto:support@getlantern.org) ]，请尽快帮助恢复，谢谢！

**Comment by casperwzy** [comment]
@Mike777Khoo ，用了好多年了，基本不需要登录，3月26日不知道什么原因，突然就变成免费版了，以前也出现过这样的情况，过几天自动就好了，帮忙看看是怎么回事吧，也没有登录界面，也显示免费版。
![360截图20240327154209545](https://github.com/getlantern/lantern-forum-cn/assets/25656932/27d30a68-eb3b-4fc3-8459-501d73ae2f74)
![360截图20240327154001642](https://github.com/getlantern/lantern-forum-cn/assets/25656932/738aca9c-10f7-4454-8411-f4fa73e5db84)

**Comment by casperwzy** [comment]
邮件有自动的回复
![360截图20240327154510443](https://github.com/getlantern/lantern-forum-cn/assets/25656932/daff6c63-58ac-470b-9813-6df1e295a777)

**Reply by MaxMKMax** [reply]
@casperwzy 给您带来的不便，敬请谅解。
想请问一下您有时看重新登陆吗? 可能是被登出了. 再登陆就可以用专业版了. 
您的账号邮箱是这个吗? 330522261@qq.com

**Reply by casperwzy** [reply, marked as answer]
是的，[330522261@qq.com](mailto:330522261@qq.com)，没有重新登录的界面，只有升级到专业版的 
![360截图20240327225834498](https://github.com/getlantern/lantern-forum-cn/assets/25656932/4a5b1b01-bc7a-45d9-adf0-6719c2926a81)
